CHANGELOG.md

```diff
@@ -3,6 +3,63 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## __VERSION__
+
+_Oct 26, 2023_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- The Tree View package is now stable!
```

Comment:
NICE ONE! 🎉

Reply:
WDYT about also highlighting the bugfixes and the #10708? 🤔

Reply:
See https://github.com/mui/mui-x/pull/10813#discussion_r1373462649